lib/vagrant-service-manager/command.rb

```diff
@@ -17,6 +16,8 @@ module ServiceManager
       'docker' => Docker, 'openshift' => OpenShift,
       'kubernetes' => Kubernetes
     }
+    # Download binary at current folder
+    BIN_FOLDER = './'
```

Comment:
not sure about that. I think the default should be something like _bin_ relative to the Vagrantfile, but maybe we want to have an additional option for the install-cli which allows to specify the target directory.